admin_guide/backup_restore.adoc

```diff
@@ -0,0 +1,197 @@
+= Backup and Restore
+{product-author}
+{product-version}
+:data-uri:
+:icons: font
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+
+// REVIEWERS: READ THIS!
+//
+// In the following text, there are questions of the form:
+//   //??? QUESTION
+// Please feel free to make a line-comment to answer them, in addition
+// to any other (line-)comments on the correctness of the text.
+//
+// - Usually, the question pertains to the text preceding it.
+//   Questions pertaining to following text are explicitly noted.
+//
+// - There are bunch of questions at the end.
+//
+// Thanks for your cooperation on this (experimental) method of
+// refining the documentation.  Hopefully it will bear good fruit.
+
+== Overview
+
+In {product-title}, you can
+_back up_, saving state to separate storage,
+and _restore_, recreating state from separate storage,
+at the cluster level.
```

Comment:
@tnguyen-rh : That's a lot of commas. Pinging :pencil2: captain grammar :memo: ( @bfallonf ) for insight. 

Maybe it's me, but I always take a pause when I hit a comma, and this many commas disrupts the flow of what you're saying. I'd suggest using brackets in place of all those commas:

In OpenShift, you can back up (saving state to separate storage) and restore (recreating state from separate storage) at the cluster level.